pkg/admin/handlers/v2/resource_handler.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package handlersV2
+
+import (
+	"net/http"
+	"sort"
+
+	"github.com/apache/dubbo-kubernetes/pkg/admin/errors"
+	"github.com/apache/dubbo-kubernetes/pkg/admin/model/req"
+	"github.com/apache/dubbo-kubernetes/pkg/admin/model/resp"
+	servicesV2 "github.com/apache/dubbo-kubernetes/pkg/admin/services/v2"
+	"github.com/gin-gonic/gin"
+	"github.com/vcraescu/go-paginator"
+	"github.com/vcraescu/go-paginator/adapter"
+)
+
+var resourceService = servicesV2.NewResourceService()
+
+func SearchApplication(c *gin.Context) {
+	// request params validation and binding
+	namespace := c.DefaultQuery(req.Namespace, "")
```

Comment:
The concept of namesapce is removed in the part of Console. Because the usage of "namespace"  can be valid among different users. So the selectors of cache should span multi namespaces.